[2024-04-22T21:33:49.867+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-22T21:33:49.892+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_cron_expression_v04.task1 scheduled__2024-04-18T03:00:00+00:00 [queued]>
[2024-04-22T21:33:49.900+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_cron_expression_v04.task1 scheduled__2024-04-18T03:00:00+00:00 [queued]>
[2024-04-22T21:33:49.901+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 6
[2024-04-22T21:33:49.911+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): task1> on 2024-04-18 03:00:00+00:00
[2024-04-22T21:33:49.917+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=3327) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-04-22T21:33:49.916+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_with_cron_expression_v04', 'task1', 'scheduled__2024-04-18T03:00:00+00:00', '--job-id', '408', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_cron_expression.py', '--cfg-path', '/tmp/tmpnnzg_8ws']
[2024-04-22T21:33:49.918+0000] {standard_task_runner.py:63} INFO - Started process 3334 to run task
[2024-04-22T21:33:49.919+0000] {standard_task_runner.py:91} INFO - Job 408: Subtask task1
[2024-04-22T21:33:49.950+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_with_cron_expression_v04.task1 scheduled__2024-04-18T03:00:00+00:00 [running]> on host e87e2f3305bd
[2024-04-22T21:33:49.994+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='julionevadod' AIRFLOW_CTX_DAG_ID='dag_with_cron_expression_v04' AIRFLOW_CTX_TASK_ID='task1' AIRFLOW_CTX_EXECUTION_DATE='2024-04-18T03:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-18T03:00:00+00:00'
[2024-04-22T21:33:49.994+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-22T21:33:50.002+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-04-22T21:33:50.002+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo dag with cron expression!']
[2024-04-22T21:33:50.006+0000] {subprocess.py:86} INFO - Output:
[2024-04-22T21:33:50.007+0000] {subprocess.py:93} INFO - dag with cron expression!
[2024-04-22T21:33:50.007+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-04-22T21:33:50.007+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-22T21:33:50.018+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=dag_with_cron_expression_v04, task_id=task1, execution_date=20240418T030000, start_date=20240422T213349, end_date=20240422T213350
[2024-04-22T21:33:50.060+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-22T21:33:50.067+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-22T21:33:50.067+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
